### DISCUSSION TITLE
Status approved but not received access to developer pack tools

### DISCUSSION THREAD

**Discussion by rohitsb20** [community/community#125600] [GitHub Education]
### Hi everyone,

Help me to to get access to developer pack tools . I have applied on 21 may 24. But till now I haven't got any email about access of tools.

**Comment by little-software-engineer** [comment]
## Access to GitHub Student Developer Pack Tools

If you have applied for the GitHub Student Developer Pack on May 21, 2024, and haven't received any email about access yet, here are steps you can take to follow up and potentially expedite the process:

### Steps to Take:

1. **Check Email Spam/Junk Folder**: 
   Sometimes, important emails might get filtered into spam or junk folders. Ensure you've checked these folders for any communication from GitHub.

2. **Verify Application Status**:
   - Go to the [GitHub Education](https://education.github.com/discount_requests/status) page.
   - Log in with your GitHub account.
   - Check the status of your application. It might still be under review.

3. **Ensure Document Quality**:
   - Ensure that the images of your school ID and acceptance letter are clear, complete, and meet GitHub's requirements.
   - If you suspect the images might be the issue, consider resubmitting them.

4. **Contact GitHub Support**:
   - If it's been a significant amount of time (usually, the review process can take up to a few weeks), you can contact GitHub support for an update on your application status.
   - You can reach out to them through the [GitHub Support](https://support.github.com/contact) page.

5. **Reapply If Necessary**:
   - If you find that there was an error in your submission or your application was rejected, you may need to correct the issues and reapply.
   - Follow the instructions provided in any rejection email or on the status page to resubmit your application correctly.


### Sample Message to GitHub Support:

```plaintext
Subject: Inquiry About GitHub Student Developer Pack Application Status

Dear GitHub Support Team,

I applied for the GitHub Student Developer Pack on May 21, 2024, but I have not yet received any email regarding the status of my application. I have checked my spam and junk folders but did not find any communication.

Could you please provide an update on the status of my application? My GitHub username is [Your GitHub Username].

Thank you for your assistance.

Best regards,  
[Your Full Name]  
[Your Email Address]  
[Your School Name]
